lib/linter/linter.js

```diff
@@ -354,14 +349,17 @@ function addProblemIfSameSeverityAndOptions(config, loc, problems, ruleId, ruleO
         return;
     }
 
+    const alreadyConfigured = existingConfigRaw
+        ? `is already configured to '${existingSeverity}'`
+        : "was not enabled";
```

Comment:
This message seems a bit ambiguous to me, maybe something like this instead?
```suggestion
        : "is not enabled so can't be turned off";
```